src/fights.ts

```diff
@@ -1392,20 +1392,26 @@ const freeRunFightSources = [
       have($skill`Comprehensive Cartography`) &&
       get("_monstersMapped") < 3 &&
       get("_VYKEACompanionLevel") === 0 && // don't attempt this in case you re-run garbo after making a vykea furniture
-      getBestFireExtinguisherZone() !== undefined,
+      getBestFireExtinguisherZone() !== undefined &&
```

Comment:
I really don't like letting this have two nullish return values; is there a reason to have these be separate returns?

Reply:
It can't be anything else unless both values are not used anymore. Undefined means not yet set, and null means deliberately set to nothing.